qt-kit/autogen.kit.d/templates/qt3d.tmpl

```diff
@@ -0,0 +1,31 @@
+# Distributed under the terms of the GNU General Public License v2
```

Comment:
For simple package we can use the common template instead like I do for llvm-kit:
https://github.com/macaroni-os/kit-fixups/blob/mark-unstable/llvm-kit/autogen.kit.d/base.yml#L7
and set *rdpend*, *depend* and *body* in this way:
```
   - qt3d:
         vars:
            rdepend: |
               app/foo
            depend: |
                app/foo2
            body: |
                src_configure() {
	           local mycmakeargs=(
	       	-DQT_FEATURE_qt3d_system_assimp=ON
                   )

	           qt6-build_src_configure
               }
```
And so remove a lot of templates files. But it's also fine in this way if you prefer.
Consider that on rdepend, depend, bdepend and body you can use the render engine too for example for SLOT ( {{ .Values.slot }})